docs/howtoguides/use_custom_apt_config.rst

```diff
@@ -0,0 +1,99 @@
+How to use the Pro Client with custom APT configuration
```

Comment:
Can we shorten this a bit to something more like "How to use custom APT configuration"?